baseball_game.py

```diff
@@ -316,6 +316,9 @@ def main():
                     cont_input = input("You win, one more(Y/N) ?")
                     if is_yes(cont_input) == True:
                         end_game = False
+                        # make random number
+                        random_number = str(get_not_duplicated_three_digit_number())
+                        print("Random Number is : ", random_number)
```

Comment:
In case of "3 strike",
1. make [new random number] - (was problem - solved!) (location of 'random_number generator' was problem)
2. continue "while True" - (too many while loop! Need a code trimming process.)